db/migrate/20210813061906_create_call_results.rb

```diff
@@ -0,0 +1,16 @@
+class CreateCallResults < ActiveRecord::Migration[5.2]
+  def change
+    create_table :call_results, id: :uuid do |t|
+      t.uuid :user_id, null: false, foreign_key: true
+      t.uuid :appointment_id, null: false, foreign_key: true
```

Comment:
My migration-fu is rusty. Does this leave enough slack for our referential integrity? ie, can we create a call result before we receive the corresponding appointment?

Reply:
Yeah, if we are restricting null here we should _not_ have the appointment be optional at the model level.

Reply:
Yup, this is how we're able to enforce presence at the db, while allowing models to sync out of order at the API level